tools/Repo-Tasks.psd1

```diff
@@ -42,13 +42,13 @@ Copyright = '(c) 2016 shahabhijeet. All rights reserved.'
 # PowerShellHostVersion = ''
 
 # Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
-# DotNetFrameworkVersion = '4.5'
+# DotNetFrameworkVersion = ''
 
 # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
-# CLRVersion = '4.0'
+# CLRVersion = ''
 
 # Processor architecture (None, X86, Amd64) required by this module
-# ProcessorArchitecture = 'x86, Amd64'

```

Comment:
@markcowl no matter how much I try, Update-ModuleManifest will not add/update None value. I have now added that manually.